README.md

```diff
@@ -1,96 +1,112 @@
+```
 
-## pipenv (recommended)
+This project uses [Pipenv](https://github.com/pypa/pipenv) for development workflow. If you don`t have it, run this command to install.
 
-簡単かつ確実なインストール方法。
 
-環境変数`PIPENV_VENV_IN_PROJECT`を1に設定すると、  
-pipenvで作成されるpythonの仮想環境がこのプロジェクトの直下に作成される。(`/path/to/HDNNP/.venv/`)  
-以下のコマンドを実行するか、`~/.bashrc`に追記してプロセスを再起動することで変更が適用される。
+**macOS**
 ```
-export PIPENV_VENV_IN_PROJECT=1
+brew install pipenv
 ```
 
+## Setup
+### By Pipenv(Prefered)
+
+Same as by anaconda, but you need to install python rather than installing anaconda. This bug will be fixed in near future release.
+
+Set environmental variable `PIPENV_VENV_IN_PROJECT` to `1` to create your VW into this project dir(`/path/to/HDNNP/.venv`).
+
+```shell
+export PIPENV_VENV_IN_PROJECT = 1
 ```
-$ git clone https://github.com/ogura-edu/HDNNP.git
-$ cd HDNNP/
-$ pyenv install 3.6.7
-$ pyenv local 3.6.7
+
+For macOS users, you need to install `mpich` before installing dependencies.
+
+```shell
+# Only for macOS users
+$ brew install mpich
+
+# Install dependencies
 $ pipenv install
 
-# activate
+# activate your VM
 $ pipenv shell
 
-(HDNNP) $ hdnnpy training
-
 # deactivate
 (HDNNP) $ exit
 ```
 
-## anaconda
 
-最適化されたバイナリを取得できるので、実行速度が速い。  
-しかし、上記の理由からpipと混在した形になることや、  
-マシンによってはインストールがうまくいかないことがあるため注意すること。
+### By Anaconda
 
-`conda env create --file condaenv.yaml`の実行が終了すると、  
-各々の環境に合わせてactivationの仕方がいくつか提示されるので好きなものを選ぶ。  
-以下の例では`~/.bashrc`に1文追記する方法を選択している。
+Using anaconda is prefered because it is basically faster than Pipenv.
 
-```
-$ git clone https://github.com/ogura-edu/HDNNP.git
-$ cd HDNNP/
-$ pyenv install anaconda-x.x.x
-$ pyenv local anaconda-x.x.x
+Install anaconda and activate your VM.
+
+```shell
+$ ANACONDA_VERSION = [YOUR_ANACODA_VERSION]
+$ pyenv install $ANACONDA_VERSION
+$ pyenv local $ANACONDA_VERSION
 $ conda env create -n HDNNP --file condaenv.yaml
-$ echo ". ${HOME}/.pyenv/versions/anaconda-x.x.x/etc/profile.d/conda.sh" > ~/.bashrc
+$ echo ". ${HOME}/.pyenv/versions/<anacondaVERSION>/etc/profile.d/conda.sh" > ~/.bashrc
 
 # activate
 $ conda activate HDNNP
 
 # install this program using pip
 (HDNNP) $ pip install --editable .
 
+# For example...
 (HDNNP) $ hdnnpy training
 
+
 # deactivate
 (HDNNP) $ conda deactivate
 ```
 
-## pip install only
+**NOTE** 
 
-`Pipfile`または`condaenv.yaml`に記述されている依存関係を元に、  
-パッケージを個別に`pip install`することももちろん可能。  
-この場合は`virtualenv`を使って自分で仮想環境を管理することを推奨する。
-```
-$ git clone https://github.com/ogura-edu/HDNNP.git
-$ cd HDNNP/
-$ pip install PKG1 PKG2 ...
-$ pip install -e .
-```
+There is no
 
-または、慣れた人であれば依存関係を`setup.py`に書き加えるだけで済む。
-```
-$ git clone https://github.com/ogura-edu/HDNNP.git
-$ cd HDNNP/
-$ vim setup.py  #=> setup()の引数にinstall_requiresを追加
-$ pip install -e .
+- ChainerMN
+- Chainer v5
+
+on the Anaconda Cloud, so you still have to install these packages by `pip`.
+
+And these is a bug that if you install anaconda by `pyenv`, `pipenv` will fail to start.
+
+## Usage
```

Comment:
thank you for your kindness, but I will add other sections in a Project(https://github.com/ogura-edu/HDNNP/projects/1), so please delete on this PR

Reply:
Gocha! Please add link to the Project when the doc about "How to use ..." is finished for better README.

Thank you